<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>OpenCobolIDE</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ide.qrc">
    <normaloff>:/ide-icons/rc/silex-64x64.png</normaloff>:/ide-icons/rc/silex-64x64.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="0" columnstretch="0">
    <property name="margin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_6">
        <property name="margin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="HomePageWidget" name="wHomePage" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="margin">
         <number>6</number>
        </property>
        <property name="spacing">
         <number>6</number>
        </property>
        <item row="0" column="0">
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>-1</number>
          </property>
          <property name="tabsClosable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBarFile">
   <property name="windowTitle">
    <string>Tollbar File</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
  </widget>
  <widget class="QToolBar" name="toolBarCode">
   <property name="windowTitle">
    <string>Toolbar Code</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionProgram"/>
   <addaction name="actionSubprogram"/>
   <addaction name="separator"/>
   <addaction name="actionCompile"/>
   <addaction name="actionRun"/>
  </widget>
  <widget class="QDockWidget" name="dockWidgetLogs">
   <property name="windowTitle">
    <string>Logs</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QTabWidget" name="tabWidgetLogs">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_3">
        <attribute name="icon">
         <iconset resource="ide.qrc">
          <normaloff>:/ide-icons/rc/applications-system.png</normaloff>:/ide-icons/rc/applications-system.png</iconset>
        </attribute>
        <attribute name="title">
         <string>Compiler</string>
        </attribute>
        <attribute name="toolTip">
         <string>Show compiler log</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QListWidget" name="listWidgetErrors"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabProgramOutput">
        <attribute name="icon">
         <iconset resource="ide.qrc">
          <normaloff>:/ide-icons/rc/media-playback-start.png</normaloff>:/ide-icons/rc/media-playback-start.png</iconset>
        </attribute>
        <attribute name="title">
         <string>Output</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QPlainTextEdit" name="plainTextEditOutput"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRecent_files">
     <property name="title">
      <string>Recent files</string>
     </property>
     <addaction name="separator"/>
     <addaction name="actionClear"/>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="menuRecent_files"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="mnuActiveEditor">
     <property name="title">
      <string>Active editor</string>
     </property>
     <addaction name="separator"/>
    </widget>
    <addaction name="mnuActiveEditor"/>
    <addaction name="actionComment_Uncomment"/>
    <addaction name="separator"/>
    <addaction name="aPreferences"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuToolbars">
     <property name="title">
      <string>Toolbars</string>
     </property>
     <addaction name="aShowFilesToolbar"/>
     <addaction name="aShowCodeToolbar"/>
    </widget>
    <widget class="QMenu" name="menuDock_panels">
     <property name="title">
      <string>Windows</string>
     </property>
     <addaction name="aShowLogsWin"/>
     <addaction name="aShowNavWin"/>
    </widget>
    <addaction name="menuToolbars"/>
    <addaction name="menuDock_panels"/>
    <addaction name="separator"/>
    <addaction name="actionFullscreen"/>
   </widget>
   <widget class="QMenu" name="menuCobol">
    <property name="title">
     <string>Cobol</string>
    </property>
    <addaction name="actionProgram"/>
    <addaction name="actionSubprogram"/>
    <addaction name="separator"/>
    <addaction name="actionCompile"/>
    <addaction name="actionRun"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuCobol"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QDockWidget" name="dockWidgetNavPanel">
   <property name="windowTitle">
    <string>Navigation</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="0">
      <widget class="QTreeWidget" name="twNavigation">
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="ide.qrc">
     <normaloff>:/ide-icons/rc/system-log-out.png</normaloff>:/ide-icons/rc/system-log-out.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Exit application (Ctrl+Q)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionCompile">
   <property name="icon">
    <iconset resource="ide.qrc">
     <normaloff>:/ide-icons/rc/applications-system.png</normaloff>:/ide-icons/rc/applications-system.png</iconset>
   </property>
   <property name="text">
    <string>Compile</string>
   </property>
   <property name="toolTip">
    <string>Compile the current file (F8)</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="ide.qrc">
     <normaloff>:/ide-icons/rc/media-playback-start.png</normaloff>:/ide-icons/rc/media-playback-start.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="toolTip">
    <string>Run the current file (F5)</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="ide.qrc">
     <normaloff>:/ide-icons/rc/dialog-information.png</normaloff>:/ide-icons/rc/dialog-information.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About OpenCobol IDE (F1)</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="ide.qrc">
     <normaloff>:/ide-icons/rc/document-save.png</normaloff>:/ide-icons/rc/document-save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save current file (Ctrl+S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="ide.qrc">
     <normaloff>:/ide-icons/rc/document-save-as.png</normaloff>:/ide-icons/rc/document-save-as.png</iconset>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="toolTip">
    <string>Save current file as (Ctrl+Shift+S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ide.qrc">
     <normaloff>:/ide-icons/rc/view-fullscreen.png</normaloff>:/ide-icons/rc/view-fullscreen.png</iconset>
   </property>
   <property name="text">
    <string>Fullscreen</string>
   </property>
   <property name="toolTip">
    <string>Toggle fullscreen (F12)</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionProgram">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ide.qrc">
     <normaloff>:/ide-icons/rc/application-x-executable.png</normaloff>:/ide-icons/rc/application-x-executable.png</iconset>
   </property>
   <property name="text">
    <string>Program</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Tells the ide to compile the current file as an executable (.exe)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSubprogram">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ide.qrc">
     <normaloff>:/ide-icons/rc/application-x-sharedlib.png</normaloff>:/ide-icons/rc/application-x-sharedlib.png</iconset>
   </property>
   <property name="text">
    <string>Subprogram</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Tells the ide to compile the current file as a subprogram (.so)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="ide.qrc">
     <normaloff>:/ide-icons/rc/document-new.png</normaloff>:/ide-icons/rc/document-new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>New file (Ctrl+N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="ide.qrc">
     <normaloff>:/ide-icons/rc/document-open.png</normaloff>:/ide-icons/rc/document-open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open a file (Ctrl+O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="ide.qrc">
     <normaloff>:/ide-icons/rc/edit-clear.png</normaloff>:/ide-icons/rc/edit-clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear list</string>
   </property>
   <property name="toolTip">
    <string>Clear list of recent files</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="aShowFilesToolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Files</string>
   </property>
   <property name="toolTip">
    <string>Show/Hide the files toolbar</string>
   </property>
  </action>
  <action name="aShowCodeToolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Code</string>
   </property>
   <property name="toolTip">
    <string>Show/Hide code toolbar</string>
   </property>
  </action>
  <action name="aShowLogsWin">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Logs</string>
   </property>
   <property name="toolTip">
    <string>Show/Hide logs window</string>
   </property>
  </action>
  <action name="aShowNavWin">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Navigation</string>
   </property>
   <property name="toolTip">
    <string>Show/Hide navigation panel</string>
   </property>
  </action>
  <action name="aPreferences">
   <property name="icon">
    <iconset resource="ide.qrc">
     <normaloff>:/ide-icons/rc/Preferences-system.png</normaloff>:/ide-icons/rc/Preferences-system.png</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
   <property name="toolTip">
    <string>Edit the application settings</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionComment_Uncomment">
   <property name="text">
    <string>Comment/Uncomment</string>
   </property>
   <property name="toolTip">
    <string>Comment/Uncomment selected/active lines</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+/</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HomePageWidget</class>
   <extends>QWidget</extends>
   <header>cobcide/home_page.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ide.qrc"/>
 </resources>
 <connections/>
</ui>
